01_fib/NicholeL/fib.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func abs(args int) (res int) {
+	if args == 0 {
+		return 0
+	}
+	if args > 0 {
+		return args
+	}
+	return args * (-1)
+
+}
+
+func fib(args int) {
+	a, b := 0, 1
+	if abs(args) > 92 {
```

Comment:
this is only true if you host system defaults to int64 for int (check the go [playground](https://play.golang.org) for a 32 bit int default).
Use explicit int64 or get rid of check (not required by lab assignment).

Reply:
will delete this check